frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditOptionChoice/EditOptionChoice.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { StudioDeleteButton } from '@studio/components';
+import { EditOptionList } from './EditOptionList';
+import { EditManualOptionsWithEditor } from '../EditManualOptionsWithEditor';
+import { OptionListEditor } from './EditOptionList/OptionListEditor';
+import type { IGenericEditComponent } from '@altinn/ux-editor/components/config/componentConfig';
+import type { SelectionComponentType } from '@altinn/ux-editor/types/FormComponent';
+import classes from './EditOptionChoice.module.css';
+
+type EditOptionChoiceProps = Pick<
+  IGenericEditComponent<SelectionComponentType>,
+  'component' | 'handleComponentChange'
+>;
+
+export function EditOptionChoice({
+  component,
+  handleComponentChange,
+}: EditOptionChoiceProps): React.ReactElement {
+  const isOptionChosen =
+    (component.optionsId !== undefined && component.optionsId !== '') ||
+    component.options !== undefined;
+  const [chosenOption, setChosenOption] = useState<boolean>(isOptionChosen);
+
+  const shouldDisplayChosenOption = isOptionChosen && chosenOption === true;
+
+  return (
+    <>
+      {shouldDisplayChosenOption ? (
+        <DisplayChosenOption
+          setChosenOption={setChosenOption}
+          component={component}
+          handleComponentChange={handleComponentChange}
+        />
+      ) : (
+        <div className={classes.optionButtons}>
+          <EditManualOptionsWithEditor
+            setChosenOption={setChosenOption}
+            component={component}
+            handleComponentChange={handleComponentChange}
+          />
+          <EditOptionList
+            setChosenOption={setChosenOption}
+            component={component}
+            handleComponentChange={handleComponentChange}
+          />
+        </div>
+      )}
+    </>
+  );
+}
+
+type DisplayChosenOptionProps = {
+  setChosenOption: (value: boolean) => void;
+} & Pick<IGenericEditComponent<SelectionComponentType>, 'component' | 'handleComponentChange'>;
+
+function DisplayChosenOption({
```

Comment:
Can we rename this component?
```suggestion
function SelectedOptionList({
```